services/distribution/src/main/resources/application.yaml

```diff
@@ -105,7 +110,7 @@ services:
         download-timeout-in-seconds: ${KEY_DOWNLOAD_ANDROID_DOWNLOAD_TIMEOUT_IN_SECONDS:30}
```

Comment:
This should default to `450`

Reply:
This is not the overall timeout just fyi. We can change it to 450, but the doc specification mentioned 30

Reply:
You're right, I did not update the spec. It should indeed be set to 450, see https://github.com/corona-warn-app/cwa-app-tech-spec/pull/6/commits/7dca7461976b2a76c1f7fcbeb4e4cc20d18416f9.